packages/web-react/src/components/Grid/demo/index.tsx

```diff
@@ -17,6 +21,18 @@ ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
     <DocsSection title="Responsive Columns">
       <GridResponsive />
     </DocsSection>
+    <DocsSection title="Custom Spacing">
+      <GridCustomSpacing />
+    </DocsSection>
+    <DocsSection title="Responsive Custom Spacing">
+      <GridResponsiveCustomSpacing />
+    </DocsSection>
+    <DocsSection title="Responsive Custom Vertical Spacing">
+      <GridResponsiveCustomVerticalSpacing />
+    </DocsSection>
+    <DocsSection title="Responsive Custom Horizontal Spacing">
+      <GridResponsiveCustomHorizontalSpacing />
+    </DocsSection>
```

Comment:
Please swap the horizontal and vertical examples so anything horizontal always goes first.